services/motion/builtin/builtin.go

```diff
@@ -37,7 +38,10 @@ func init() {
 			) (motion.Service, error) {
 				return NewBuiltIn(ctx, deps, conf, logger)
 			},
-			WeakDependencies: []internal.ResourceMatcher{internal.SLAMDependencyWildcardMatcher},
+			WeakDependencies: []internal.ResourceMatcher{
+				internal.SLAMDependencyWildcardMatcher,
+				internal.ComponentDependencyWildcardMatcher, // once ResourceMatchers are more built out this should only match bases
```

Comment:
Why not all actuators?

Reply:
I did this as a temporary measure because the way that "Matchers" work at the moment is that actually they're hardcoded and it was easier for me to grab an existing one than create a new type.  But I can do that here since you're right and this would be more accurate.

Reply:
We are doing this in a really weird way currently.  I made RSDK-3175 to track this.

Reply:
Thanks! Reading the ticket, it sounds like we might have the same problem for bases once we want to relate the base to the slam component reference through the lidar, so we might have to get the base directly from the frame system. Is that correct?

Reply:
This is not a bridge I had tried to cross yet but at first glance I think you are right that this will pose a similar problem.  There is a workaround which is calling the frame system function `CurrentInputs` to get all the resources but this is something we'll want to phase out

Reply:
Ok cool